dependencywalker.txt

```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = Dependency Walker
+Description = A free utility that scans any 32-bit or 64-bit Windows module (exe, dll, ocx, sys, etc.) and builds a hierarchical tree diagram of all dependent modules.
+LicenseType = 2
+Category = 7
+URLSite = https://dependencywalker.com
+Version = 2.2
+URLDownload = https://dependencywalker.com/depends22_x86.zip
+SizeBytes = 468618
+Installer = Generate
+
+[Section.amd64]
+URLDownload = https://dependencywalker.com/depends22_x64.zip
```

Comment:
No SizeBytes for the x64 entry?

Reply:
Also we usually add a SHA entry to verify the download has been done correctly. This seems missing.